tests/files/rbdigital/response_patron_checkouts_200_list.json

```diff
@@ -8,55 +8,55 @@
         "id": 1646766,
         "display": "Introduction",
         "filename": "78226_001_IN.mp3",
-        "downloadUrl": "http://api.oneclickdigital.us/v1/media/9781456103859/parts/1646766/download-url?s3=78226&f=78226_001_IN",
+        "downloadUrl": "http://api.rbdigital.us/v1/media/9781456103859/parts/1646766/download-url?s3=78226&f=78226_001_IN",
```

Comment:
It's okay to change this, and we don't use these URLs, but to be accurate it should be changed to api.rbdigital.com